### PR TITLE
Style check

### PR DESCRIPTION
**Pull request by moshekruger**

### CODE REVIEW

File: docs/learn-about-neptune/team-management.rst [hunk 1]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
...
    :alt: Add users to project
 
-| If it is project created in team organization, user is assigned to project immediately.
-| If it is project created in the individual account, user is invited to the project via mail.
-  User needs to accept invite, before you can see such person on the list above.
+
+.. note::
+
+    - If the project is created in a team organization, the user is assigned to the project immediately.
+    - If the project is created in an individual account, the user is invited to the project by email.
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: You cannot invite users to projects in individual organizations.

File: docs/learn-about-neptune/team-management.rst [hunk 2]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
...
+    - The invited user must accept the invitation in order to appear in the list of project members.
```
Comment (pitercl) [inline docs/learn-about-neptune/team-management.rst]: I don't think that's true for projects in team organizations. I think it's more like an assignment.

User only need to accept invitations to team organizations.

File: docs/python-api/how-to/track.rst [hunk 3]
```diff
@@ -215,26 +216,26 @@ All you need to do it to pass empty list ``[]`` to the ``upload_source_files`` p
...
-Go to your Jupyter, where you can see two Neptune buttons:
+Go to your Jupyter, where you see two Neptune buttons:
 
-* **n** button is for configuration changes
-* **Upload** button is for making checkpoint in Neptune
+* **Connect to Neptune** is for configuration changes.
+* **Upload** is for making a checkpoint in Neptune.
 
 .. image:: ../../_static/images/notebooks/buttons_02_1.png
    :target: ../../_static/images/notebooks/buttons_02_1.png
```
Comment (pitercl) [inline docs/python-api/how-to/track.rst]: This is an old screenshot.

File: docs/python-api/how-to/track.rst [hunk 4]
```diff
@@ -215,26 +216,26 @@ All you need to do it to pass empty list ``[]`` to the ``upload_source_files`` p
...
    :alt: image
 
-Click **Upload**, whenever you want to create new checkpoint in Neptune. You will see tooltip with link as a confirmation.
+Click **Upload** whenever you want to create a new checkpoint in Neptune. 
+A notification containing a link appears as confirmation.
 
 .. image:: ../../_static/images/notebooks/buttons_03_1.png
    :target: ../../_static/images/notebooks/buttons_03_1.png
```
Comment (pitercl) [inline docs/python-api/how-to/track.rst]: This is an old screenshot.

File: docs/python-api/how-to/track.rst [hunk 5]
```diff
@@ -215,26 +216,26 @@ All you need to do it to pass empty list ``[]`` to the ``upload_source_files`` p
...
-Upload notebook checkpoint
+Upload Notebook checkpoint
 --------------------------
-I want to add Notebook checkpoint to my project. How to do it?
+I want to add a Notebook checkpoint to my project.
 
 Solution
 ^^^^^^^^
-Go to your Jupyter, where you can see two Neptune buttons:
+Go to your Jupyter, where you see two Neptune buttons:
```
Comment (pitercl) [inline docs/python-api/how-to/track.rst]: Not 2 buttons any more.

File: docs/python-api/how-to/track.rst [hunk 6]
```diff
@@ -187,7 +188,7 @@ Calculate the hash of your training data and send it to Neptune as text:
 
 **Step 3**
 
-Add data version column to your project dashboard:
+Add a data version column to your project dashboard:
 
 .. image:: ../../_static/images/how-to/ht-data-version-1.png
```
Comment (pitercl) [inline docs/python-api/how-to/track.rst]: An old screenshot.

File: docs/learn-about-neptune/core-concepts.rst [hunk 7]
```diff
@@ -1,116 +1,164 @@
...
 
-Organization is a way to centrally manage projects, users (and billing). Neptune has two organization types.
+Neptune has two organization types: Individual and Team
 
-Individual
-^^^^^^^^^^
+Individual organizations
+""""""""""""""""""""""""
 * Each user is assigned individual organization with ``username`` as an organization name.
 * User is the only member of this organization but may :ref:`invite collaborators <how-to_team-management_invite-to-project>` to projects.
```
Comment (pitercl) [inline docs/learn-about-neptune/core-concepts.rst]: Not true.

File: docs/learn-about-neptune/core-concepts.rst [hunk 8]
```diff
@@ -1,116 +1,164 @@
...
 
-Team
-^^^^
-* Team organization comes handy when entire team needs to be managed centrally.
-* Once :ref:`created <how-to_team-management_create-organization>`, team organization can be managed by :ref:`organization Admin <core-concepts_user-roles_organization-admin>`. This include users and billing.
-* Only Users who joined team organization can browse its content, subject to assigned :ref:`role <core-concepts_user-roles>` in the organization or project.
+Team organizations
+""""""""""""""""""
+* Team organizations comes in handy when an entire team needs to be managed centrally.
+* Once :ref:`created <how-to_team-management_create-organization>`, a team organization can be managed by :ref:`organization Admin <core-concepts_user-roles_organization-admin>`. This include users and subscription.
```
Comment (pitercl) [inline docs/learn-about-neptune/core-concepts.rst]: This includes

File: docs/learn-about-neptune/core-concepts.rst [hunk 9]
```diff
@@ -1,116 +1,164 @@
...
-    <a href="https://ui.neptune.ai/neptune-ai/credit-default-prediction" target="_blank">Credit default prediction</a>
+**Admin**
 
+.. note::
 
-.. |binary-classification-metrics| raw:: html
+    An organization must have at least one Admin.
 
-    <a href="https://ui.neptune.ai/neptune-ai/binary-classification-metrics" target="_blank">Binary classification metrics</a>
+By default, an Admin is the Owner of all projects (this setting is editable).
```
Comment (pitercl) [inline docs/learn-about-neptune/core-concepts.rst]: I don't believe it is editable.

File: docs/learn-about-neptune/core-concepts.rst [hunk 10]
```diff
@@ -1,116 +1,164 @@
...
+
+* Has edit access to all experiments and Notebooks.
+* Has edit access to project settings.
+* Can remove projects.
+* A project creator is by default the project Owner.
+
+**Contributor**
+
+* Has edit access to his or her own experiments and Notebooks.
+* Has view access to all his or her experiments and Notebooks.
```
Comment (pitercl) [inline docs/learn-about-neptune/core-concepts.rst]:
```
* Has edit access to his or her own experiments and Notebooks.
* Has view access to all his or her experiments and Notebooks.
```
Contributors have edit access similar to Owners.

File: docs/python-api/how-to/track.rst (hunk 5 above)
Comment (moshekruger) [reply docs/python-api/how-to/track.rst]: done

File: docs/learn-about-neptune/core-concepts.rst (hunk 8 above)
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst (hunk 9 above)
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: OK.

File: docs/learn-about-neptune/core-concepts.rst (hunk 10 above)
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: OK

File: docs/learn-about-neptune/team-management.rst [hunk 11]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
 
 .. _how-to_team-management_create-organization:
 
-Create organization
+Create an organization
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: OK

File: docs/learn-about-neptune/team-management.rst (hunk 1 above)
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: We discussed.

File: docs/learn-about-neptune/team-management.rst (hunk 2 above)
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: Discussed

File: docs/python-api/how-to/track.rst (hunk 3 above)
Comment (moshekruger) [reply docs/python-api/how-to/track.rst]: replaced

**Pull request closed without merging**